### PR TITLE
Add a troubleshoot document for HPU

### PR DESCRIPTION
**Pull request by jerome-habana**
Signed-off-by: Jerome <janand@habana.ai>

## What does this PR do?

Fixes #\<issue_number>

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [ ] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [ ] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: docs/source-pytorch/accelerators/hpu_troubleshoot.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+**Audience:** Gaudi chip users looking to troubleshoot their model changes during training with HPU.
+
+----
+
+Runtime Errors
+--------------
+
+This section provides troubleshooting instructions that can be referred to for common issues when training PyTorch models. The following are common functional issues that may occur when running on HPU and not on CPU/GPU.
+
+The following table outlines some possible runtime errors:
```
Comment (kaushikb11) [inline docs/source-pytorch/accelerators/hpu_troubleshoot.rst:16]: @jerome-habana Thanks for starting a troubleshooting doc.

The Table with solutions doesn't seem specific to the HPU Accelerator, as it is generic to all accelerators.

Comment (jerome-habana) [reply docs/source-pytorch/accelerators/hpu_troubleshoot.rst:16]: I agree @kaushikb11, we need a mechanism for users who search with the error codes to get resolution. As of now, I dont see any documentation that aids with it

File: docs/source-pytorch/accelerators/hpu_troubleshoot.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
++----------------------------------+----------------------------------+
+| Error                            | Solution                         |
++==================================+==================================+
```
Comment (awaelchli) [inline docs/source-pytorch/accelerators/hpu_troubleshoot.rst:20]: The checks whether HPU devices are available etc. can be put into the accelerator connector (we should already have it). The error messages can be adjusted to give clear information and instructions to the user (I believe we already have this, but can be improved if needed). A troubleshooting guide like the one in this table should not be needed. I strongly advise against it, and instead resolve all issues directly through good error messages.

**Review by awaelchli — CHANGES_REQUESTED** [review]

**Comment by jerome-habana** [comment]
Closing as the maintainers dont want separate troubleshoot document

**Pull request closed without merging**